### PR TITLE
support onlookup hook

### PR DESCRIPTION
**Pull request by mafintosh**
useful for debugging and needed for a future optimization where peers can share the path to a key

### CODE REVIEW

File: lib/get.js [hunk 1]
```diff
@@ -211,3 +222,7 @@ function allDeletes (list) {
 function finalize (req) {
   req._finalize()
 }
+
+function lookupAll (fn, ptrs) {
```
Comment (andrewosh) [inline lib/get.js]: lookupAll -> mapPointers

**Comment by mafintosh** [comment]
@andrewosh PTAL

**Comment by andrewosh** [comment]
:+1:

**Comment by mafintosh** [comment]
3.1.0

**Pull request merged**